### DISCUSSION TITLE
Global radiative fraction defaults

### DISCUSSION THREAD

**Discussion by johodges** [firemodels/fds#13114] [General]
Some of my colleagues were discussing how they handle RADIATIVE_FRACTION in PBD in their regions. The issue they run into is that the local authorities want designers to follow BR 368 which species a radiative fraction as part of the recommendations. When they run a case with the default FDS parameters, the global radiative fraction does not match the user prescribed. 

I looked into their case in a bit more detail and wanted to start a discussion here. The attached cases compare a slightly modified version of the Verification/Pressure_Solver/hallways.fds case. The modifications were to expand the range on QR_CLIP, C_MIN, and C_MAX such that the global radiative fraction should match the user specified and compare these with the FDS defaults. Initially, the global radiative fraction for all 3 cases matches the specified value for propane at 0.29. The cases start to diverge much more quickly than I was expecting before running the simulations.

This is not a bug since FDS is doing what the design documentation says it is doing. The question I have is whether the design meets the intended objective of matching the global radiative fraction. It seems unintuitive to me to have C_MIN < 1 if the goal of RADIATIVE_FRACTION is to account for unresolved flames. Thoughts?

![image](https://github.com/firemodels/fds/assets/17127980/a4e6265b-c670-4b41-8558-fbde2ab19fde)
[hallways.zip](https://github.com/user-attachments/files/16070928/hallways.zip)

**Comment by drjfloyd** [comment]
Are your colleagues / the AHJ correctly interpretting the intent of BR 386?  The only meaningful constant value for a radiant fraction would be from the flame volume. A global fraction including all impacts from the plume / hot surfaces / etc. cannot be a prescribed constant as the details of confinement and relative fire size would start to matter.

**Comment by johodges** [comment]
I have not talked with the AHJ, so my knowledge on their understanding is second hand. The requirement comes from the statement below. This looks to be in line with specifying a global Q_RADI/HRR. My thought is that the BR 386 guidance is tailored more to empirical and zone model calculations since it was published in 1999 before CFD fire modeling was widely used.

"Table 3.3 summarizes several of the more commonly adopted steady-state design fire sizes in current use. The heat release rate (qfAf) is the total heat generated by combustion per second, and is the parameter measured in most calorimetry experiments. Some of this heat is
radiated from the flames, and warms the surrounding walls, floor, etc. The heat remaining in the gases is the convective heat flux, and is the heat flow parameter required for calculations of subsequent smoke movement."

**Reply by drjfloyd** [reply]
"some of this is heat is radiated from the flames" That text to me as referring to the radiant output of the flame volume. If you start creating a hot, smoky layer, then that radiant output will add to that of the fire. This is not an effect that any of the simple plume correlation based design methods account for.  They just want to know the convective fraction from the top of the fire and use that to estimate the plume flow rate at various heights. CFAST would to the extent it radiates from the hot layer to the floor.

**Comment by obscureed** [comment]
Maybe we should clarify that BR 368 is a 1999 document, "Design methodologies for smoke and heat exhaust ventilation", published by CRC Ltd with the permission of the UK's Building Research Establishment.

As far as I can tell, that document does not actually specify a radiative fraction -- is that correct? What it does is specify values of "Total convective heat flux" for some official or semi-official design fire sizes, which are clearly larger. The difference between the total fire size and the convective fire size is not discussed, but it is presumably a radiative fraction in some sense. This approach is clearly not thinking about fire chemistry or even plume size in any detail at all -- so the convective heat flux for a sprinklered hotel room could be 400kW "close to the plume" or 300kW "at the window", out of a total of 500kW (deduced from fire area 2 m^2 and heat release rate density 250 kW/m^2). I don't know where the missing 100kW or 200kW is supposed to have gone.

I agree with the idea that BR368 quoted these values for use in zonal models and not in CFD. The design fire sizes were copied in British Standard BS7346-4 in 2003 (and still valid), but the column of convective fire sizes was omitted.

I have always hoped that the RADIATIVE_FRACTION parameter in FDS is not critical for my simulations, which almost always involve large fires and appreciable soot fractions. The path length for radiation anywhere near the fire is much less than 1m, so I have assumed that convection and radiation would reach equilibrium with each other within a few metres (and probably a lot sooner). The RADIATIVE_FRACTION parameter is effectively a starting guess in this equilibrium. If a different value was specified, I would expect to see a somewhat different flame temperature, and hence probably a slightly different plume size, but the same amount of heat has to leave the fire bed, and it will leave in more or less the same way. Is this a fair summary? (I know, I should check the sensitivity in my models -- but it's not at the top of the list of sensitivities to investigate.)

**Reply by drjfloyd** [reply]
The convective fraction is 1 - the radiative fraction. With the default value of 0.35 this is an 0.65 convective fraction. For a smoke control system design, this drives the volume flow rate you need to extract to maintain the layer above some height. But if your true radiant fraction was 0.40 or 0.30, then your convective fraction would be 0.60 or 0.70. A 10 % shift.  The volume flow would scale with the change in temperature rise. 10 % more energy into the layer would mean 10 % more temperature rise. With a smoke control system you can't have too hot of a layer otherwise it would radiate enough heat to cause burns to occupants. A 200 K temperature rise in the layer would make for unfriendly conditions for egress. 10 % of 200 K would be 20 K or a change in density (and hence volume flow) of just a few percent.

**Reply by johodges** [reply]
The radiative fraction is specified implicitly through the total heat release rate (specified fire area and HRRPUA) and the specified total convective heat flux. The retail scenarios specify 20% for retail areas (5000 kW convective / (10 m2 x 625 kW/m2 total)), 34% for open-plan offices, 25% for car parks.

I agree for smoke control design it probably will not make too much of a difference in the calculation. However, we also use the global RADIATIVE_FRACTION to correct the local heat fluxes from the fire in the code. How we handle RTE_CORRECTION_FACTOR can have a pretty big impact on these heat fluxes. In the figure below, I shifted the fire to be in the corner in the hallway case and I am comparing the GAUGE HEAT FLUX on the vertical wall in the three cases.

Consider the case where the hallway is infinitely straight. If we only mesh an area fairly close to the fire, RTE_CORRECTION_FACTOR will be higher. As we extend the mesh further down the hallway, RTE_CORRECTION_FACTOR will continue to decrease until it hits C_MIN (1.0 as the default in FDS). This behavior makes sense with RADIATIVE FRACTION being defined on a global basis. However, we run into issues when we apply that global value to try and better resolve local unresolved flames. In the limit that we extend the domain until RTE_CORRECTION_FACTOR=C_MIN, we are no longer applying any local correction in the flaming region.

![image](https://github.com/firemodels/fds/assets/17127980/1f3621ba-622f-4c43-a6ed-8d982a583f9a)

**Comment by mcgratta** [comment]
I don't follow the reasoning as to how the `RTE_CORRECTION_FACTOR` approaches `C_MIN` as the domain gets larger. Are we assuming that the combustion region of the fire is completely within the mesh?

**Comment by drjfloyd** [comment]
The whole concept of radiant fraction as commonly used in fire protection is based on measurements made of fires in the open. That 35 % radiant fraction does not apply as a global value once you start confining the fire and the fire plume.  I don't think it is appropriate in a design application to set QR_CLIP = -1 so the RADIATIVE_FRACTION is treated as global as there is no way you would know the global fraction for a specific geometry a priori. In reality, when you add more and more hallway length, the fire is going to radiate the same nominal amount as when it was in the open and as you increase the hallway length and have more and more length of upper layer the loss from the upper layer will increase. Overall the global value increases but the fire is going to stay more or less the same. Setting QR_CLIP = -1 has two large issues.  1) To get the correct radiant source term as the real fire you would need to up the RADIATIVE_FRACTION but you don't know by how much. 2) The reason we need correction is we don't resolve the 2000 K flamelet temperature on an egineering grid. At most we get up to 1000-1200 K.  In the layer down the hallway. however, that temperature is a good temperature for the source term from that gas. There is no unresolved flame down the hall. Correcting the flame volume and the hall layer with the same factor would not be physically valid.

**Reply by johodges** [reply]
Agreed that we do not want to set QR_CLIP=-1 when we are looking at local flames. There is some discussion on this topic in the user's guide on large tank fires where the radiative fraction can get pretty low due to the development of a soot mantle. In this case setting QR_CLIP=-1 may be appropriate since the goal would be to have the global radiative fraction match the empirically derived geometric specific global fraction. However, doing so changes the physical interpretation of radiative fraction such that it is no longer representative of unresolved flames.

**Comment by mcgratta** [comment]
Now I understand where Jonathan is coming from. The `QR_CLIP` defines the region from which we want to have a net emission of the `RADIATIVE_FRACTION`. It is meant to be something like the "flame envelop", not the entire domain. As for `C_MIN`, we don't want to "correct" the radiative emission if the grid gets finer and finer. In other words, we increase the net emission on coarse grids, typically, but we don't want to adjust it on fine. That's the idea, anyway.

**Reply by johodges** [reply]
That's a good way of summarizing it. However, I agree that I was wrong in my prior post and RTE_CORRECTION_FACTOR shouldn't converge to C_MIN as we increase the length of the hallway. I was thinking RTE_CORRECTION_FACTOR was calculated based on all cells (i.e., QR_CLIP=-1), but then by default it was only applied to cells where RADIATIVE_FRACTION x HRRPUV > 10. However, I just checked radi.f90 and that is not the case. It does look like it is inversely correlated with length up until the ~48m at which it is converged with the 96m case as shown in the first figure below. I am not sure why there would be an impact with length since there isn't additional combustion occurring further away (the global HRR is converged even at the 3m case).

Agreed with Jason's point that the losses to radiation should increase with the size of the domain due to more losses accumulating as you get further from the fire. The second figure below shows this impact for a straight hallway at a few lengths. I think the confusion is more about the interpretation and communication of what Q_RADI / HRR in the output file means.

![image](https://github.com/firemodels/fds/assets/17127980/ca9f3f66-9f72-428a-923c-2bc769e135bc)

![image](https://github.com/firemodels/fds/assets/17127980/929f1d39-06ea-42ed-a238-59ca371ff3f3)

**Reply by drjfloyd** [reply]
The correction factor is CHI_R*Q / SUM(KFST4).  So reducing the factor means either Q is lower or KFST4 is higher.  Q is the same, so roughly species in the plume (i.e. K) are going to be similar, so a reduction implies an increase in T. There will be some mixing at the interface of the outflowing and inflowing air in the hallway. More length gives more time for mixing.  Since KSFT4 is T^4, to go from a factor ~1.4 at 3 m to ~1.1 at  96 m would only require a 6 % increase in the temperature in the flame volume.  That doesn't seem like an implausible amount over 96 m of mixing bewtween layers.

Looking at 14.1, I think we could be more precise in our defintion. I can see how that could be interpretted as being a global fraction.

**Comment by obscureed** [comment]
I think there should be a distinction between your developer-level discussion (which I do not completely follow) and what users should be adjusting (which is how the original question was posed). For modelling smoke in hallways, I do not think it is necessary or appropriate to adjust C_MIN, C_MAX or QR_CLIP. I do not see any requirement to match some convective/radiative info in the BR 368 document -- whatever results come out of the FDS model are likely to be more realistic than a pre-specified fraction. None of the validation cases alter those three parameters, except the McCaffrey plume set, which concerns radiation measurements from soot-free flames. I'm approaching this from the perspective of models where the flames are optically thick, and heat loss to walls near the flame is not a critical result -- in that context, I don't see any need at all to tinker with those settings. Please correct me if I'm wrong on this.

**Reply by drjfloyd** [reply]
For typical engineering applications of FDS, C_MIN, C_MAX, and QR_CLIP would not be changed. As you note, RADIATIVE_FRACTION might change if you had some knowledge of the fuel (like a lower fraction for METHANE or METHANOL).

**Reply by johodges** [reply]
Agreed with this assessment. I wouldn't recommend changing any of these parameters in a design configuration unless the user understands how they are affecting the underlying assumptions made in FDS and those changes in assumptions align with the design objectives. 

I have been looking into this in a bit more detail relating to tunnel fires in particular. Most of the existing guidance is based on zone fire modeling configurations and usually just mentions "heat release rate" without clarifying between convective and total. Unfortunately, this is a pretty common omission in test reports as well.
